doc/source/getting_started/introduction.ipynb

```diff
@@ -83,17 +89,17 @@
     "\n",
     "\n",
     "\n",
-    "## Simple methods\n",
+    "## Simple accessors and methods\n",
     "\n",
     "Now we have our `GeoDataFrame` and can start working with its geometry. \n",
     "\n",
-    "Since we have only one geometry column read from the file, it is automatically seen as the active geometry and methods used on `GeoDataFrame` will be applied to the `\"geometry\"` column.\n",
+    "Since there was only one geometry column in the New York Borughs dataset, this column automatically becomes the _active_ geometry and spatial methods used on the `GeoDataFrame` will be applied to the `\"geometry\"` column.\n",
```

Comment:
```suggestion
    "Since there was only one geometry column in the New York Boroughs dataset, this column automatically becomes the _active_ geometry and spatial methods used on the `GeoDataFrame` will be applied to the `\"geometry\"` column.\n",
```

typo